.github/pull_request_template.md

```diff
@@ -1,41 +1,8 @@
 ## Checklist before requesting a review
 - [ ] I have performed a self-review of my code
 - [ ] Does the code follow the team's coding standards and best practices?
-- [ ] Is the code well-organized and easy to understand?
-- [ ] Are file, variable and function names clear and meaningful?
 - [ ] Are there any duplicated code blocks that can be refactored into reusable functions?
-
-## Functionality
+- [ ] Are there tests on the new code?
 - [ ] Does the PR implement the intended functionality or fix the issue described in the title and description?
-- [ ] Are there any edge cases that have been overlooked?
-
-## Testing
-- [ ] Are there sufficient tests covering the changes?
-- [ ] Are there any missing test cases that need to be added?
-- [ ] Has the code been tested in different environments if applicable (e.g., browsers, platforms, devices)?
-
-## Documentation
-- [ ] Are the changes well-documented and easy to understand?
-- [ ] Are there any missing or outdated comments in the code?
 - [ ] Is the PR title clear and detailed enough to provide necessary context?
-
-## Performance
-- [ ] Do the changes have any impact on the system's performance?
-- [ ] Are there any potential bottlenecks that need to be addressed?
-
-## Dependencies
-- [ ] Have the changes introduced any new dependencies or updated existing ones?
-- [ ] Are the dependencies up-to-date and secure?
-
-## Compatibility
-- [ ] Will the changes have any adverse effects on other parts of the system?
-- [ ] Do they work well with other existing features?
```

Comment:
I would include at least one of these questions. This is something we need to think about and may not always be at the forefront of developers' minds.